src/__tests__/lib/containers/download_list_v2/DownloadListTableV2.test.tsx

```diff
@@ -88,16 +85,46 @@ describe('DownloadListTableV2 tests', () => {
         pageParams: [],
       },
       fetchNextPage: mockFetchNextPage,
-      hasNextPage: true,
+      hasNextPage: false,
       isLoading: false,
       isSuccess: true,
     })
-
+  })
+  it('loads more available download files when inView', async () => {
+    mockAllIsIntersecting(true)
+    
     renderComponent()
 
     const fileEntity1 = await screen.findAllByText('file1.txt')
     expect(fileEntity1).toHaveLength(1)
     const fileEntity2 = await screen.findAllByText('file2.txt')
     expect(fileEntity2).toHaveLength(1)
   })
+  describe("Copy all Synapse IDs", () => {
+    const originalClipboard = { ...global.navigator.clipboard }
+    afterEach(() => {
+      Object.assign(navigator, {
+        clipboard: originalClipboard,
+      })
+    })
+
+    it("should call clipboard.writeText with the expected Synapse IDs", async () => {
+      const mockWriteText = jest.fn()
+      mockWriteText.mockResolvedValue('copied')
+      const mockClipboard = {
+        writeText: mockWriteText,
+      };
+      Object.assign(navigator, {
+        clipboard: mockClipboard,
+      })
+
+      renderComponent()
+      
+      const copySynIDsButton = await screen.findByTestId('copySynIdsButton')
+      userEvent.click(copySynIDsButton)
+      await screen.findAllByText('file2.txt')
```

Comment:
Not clear to me what this line is checking. Is it testing that a second page was fetched? That the blocking loader is gone? Not sure it would actually work in the blocking loader case, RTL can probably always see that text if it's already there.

Reply:
Unclear to me too, removed!